iroh/tests/cmd/lookup.trycmd

```diff
@@ -1,5 +1,10 @@
 ```
 $ iroh p2p lookup 1AXRDqR8jTkwzGqyu3qknicAC5X578zTMxhAi2brppK2bB
 peer id: 1AXRDqR8jTkwzGqyu3qknicAC5X578zTMxhAi2brppK2bB
+listening addresses: []
```

Comment:
It would be nice to have a fixture and test that shows a more involved lookup result so we can examine its formatting.